### DISCUSSION TITLE
Double click to open in new tab

### DISCUSSION THREAD

**Discussion by wearereact** [protz/thunderbird-conversations#1617] [Q&A] [answered]
Hello, before anything I just wanna thank you for this awesome add-on that improves Thunderbird so much.

My problem is that I stupidly did an update to a beta version (87.0b3) and, when I have this add-on activated, the click to open in new tab email doesn't work. Any thoughts on how I can get back to have this add-on and the beta version working together?

Rolling back to Thunderbird 78 doesn't work since the old profile with all my information cannot be read..

Thanks in advance,
Hugo

**Comment by Standard8** [comment, marked as answer]
Thank you for the report, I see the issue with opening in a new tab in the latest beta, hopefully I'll have a fix in a few days.

Rolling back to Thunderbird 78 is obviously not recommended - the profile data has changed in a non-backwards compatible way since then, so rolling back may loose data one way or another. Once the next major version of Thunderbird is released, you should be able to switch back across to that - you probably want to stop your beta updates on TB 91beta, then install the release version once it is out (it can take a few weeks after the beta).

**Reply by wearereact** [reply]
Thank you so much!

**Comment by NaveenGarg** [comment]
Greetings! Indeed, I love this add-in with my latest release version of Thunderbird 91.2.1. I however have the same issue. Double Click now only works if set to open in a new tab. If I set the double-click to open in a new window (my preference), double-click does not do anything unfortunately. Please let me know if and when this issue is resolved. - Thank you!

**Reply by Standard8** [reply]
Double click to open in either a new tab or a new window is working fine for me. It does ignore the existing window setting, but that doesn't seem to be what you're saying.

Try checking other add-ons aren't getting in the way.

If that fails, go to the three bar menu -> Developer Tools -> Error Console and then see if anything appears when you try and open a message with the preference set.

**Reply by NaveenGarg** [reply]
Thanks for prompt response. I disabled a couple of other add-ons (spell checker, highlighter) but left the mail sync ones (no UX) and then I turned on the Error Console as you recommended and here is what I get:

Uncaught (in promise) Error: Invalid tab ID: -1
normalizeError resource://gre/modules/ExtensionCommon.jsm:755
wrapPromise resource://gre/modules/ExtensionCommon.jsm:903
undefined

**Reply by Standard8** [reply]
To be honest, I'm not sure what's happening here. If opening in a new window is selected, then I'm not sure how you'd be triggering something that's checking the tab id.

**Reply by NaveenGarg** [reply]
I also tried changing the settings using the Config Editor but with same results. However I found there seem to be 3 related settings as below:
mail.openMessageBehavior 2
mail.openMessageBehavior.version 1
mail.openMessageBehaviour true

I have tried various combinations and values for the above 3 settings and the only setting that triggers double-click response is when the first one above is set to 2 - ie open message in a new tab. When set to 0 to open in new window instead, it returns above uncaught error (Invalid tab ID). I tried changing the version number to 0, 1, 2, 3 etc and the flag to true or false - but didn't find any combination that worked.
